### PR TITLE
add marketplace-builder to builder-docker-images scripts

### PR DESCRIPTION
**Pull request by imabdulbasit**

### CODE REVIEW

File: process-compose.yaml [hunk 1]
```diff
@@ -490,7 +490,6 @@ processes:
       failure_threshold: 100
 
   marketplace-reserve-builder:
-    disabled: true
```
Comment (QuentinI) [inline process-compose.yaml:493]: This (and removing `[marketplace]` in `docker-compose.yml`) will cause conflicts, as both services bind to `ESPRESSO_BUILDER_SERVER_PORT`.

Comment (imabdulbasit) [reply process-compose.yaml:493]: do we want to run both builders at the same time or only one of those?

Comment (imabdulbasit) [reply process-compose.yaml:493]: fallback has a separate port in .env file?

Comment (QuentinI) [reply process-compose.yaml:493]: We want to run either both marketplace builders when we're upgraded to marketplace or one legacy builder otherwise

Comment (imabdulbasit) [reply process-compose.yaml:493]: ok I will revert it for now

**Pull request merged**